game/src/edit/mod.rs

```diff
@@ -854,7 +855,8 @@ impl ConfirmDiscard {
             discard,
             panel: Panel::new(Widget::col(vec![
                 Widget::row(vec![
-                    Widget::draw_svg(ctx, "system/assets/tools/alert.svg")
+                    Image::untinted("system/assets/tools/alert.svg")
```

Comment:
<img width="181" alt="Screen Shot 2021-02-25 at 4 49 11 PM" src="https://user-images.githubusercontent.com/217057/109239039-76a9da80-7789-11eb-81a6-5d944437426c.png">

In almost all cases, the image is an icon template which we want to re-color to match the theme, but in just a *few* places, we don't want to touch the image colors, hence `untinted`.